### PR TITLE
refactor operation nodes so they don't interfere with layout

### PR DESCRIPTION
**Pull request by jacobheric**
So, in order to try out more layout frameworks, I had to refactor how we handle our operation nodes. Previously they were first class react flow nodes, yet we rendered them as children of the group nodes. This was causing issues with the frameworks. So I've pulled them out simply rendered them as content on the group nodes. 

I also went ahead and made a bunch of layout improvements that will hopefully tide us over until we get a framework working.

Here's how those improvements handle some of the slightly more complicated maps that were previously broken:

Screenshot 2023-09-06 at 1 54 05 PM
Screenshot 2023-09-06 at 1 51 29 PM
Screenshot 2023-09-06 at 1 47 59 PM

### CODE REVIEW

File: components/icons/crud.tsx [hunk 1]
```diff
@@ -1,109 +1,109 @@
 export const Edit = (props: any) => {
```
Comment (jacobheric) [inline components/icons/crud.tsx:1]: Sorry unrelated changes in this file that the linter made.

**Comment by dselans** [comment]
This is awesome!!!!

**Review by blinktag — APPROVED** [review]
